src/main/java/attendanceProject/controller/StudentViewController.java

```diff
@@ -0,0 +1,41 @@
+package attendanceProject.controller;
+
+import attendanceProject.domain.AttendanceRecord;
+import attendanceProject.domain.CourseOffering;
+import attendanceProject.service.attendanceService.AttendanceService;
+import attendanceProject.service.courseOfferingService.CourseOfferingService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Objects;
+
+@RestController
+@RequestMapping("/student-view")
+public class StudentViewController {
+    @Autowired
+    private AttendanceService attendanceService;
+
+    @Autowired
+    private CourseOfferingService courseOfferingService;
+
+    @GetMapping("/course-offerings/{offeringId}/attendance")
+    public ResponseEntity<?> getCourseOfferingAttendance(@PathVariable Long offeringId, @RequestParam Long studentId) {
+        CourseOffering courseOffering = courseOfferingService.getCourseOfferingById(offeringId);
```

Comment:
i think here ,we can have a controller for courseoffering so we can make a http call using webflux or the thing to make async http , but right now we keep it as it is